examples/yaw/cube_in_a_box_domino.clj

```diff
@@ -0,0 +1,171 @@
+
+(ns yaw.cube-in-a-box-domino
+  "A simple 3D example using a reagent-like approach."
+  (:require 
+   [clojure.set :as set]
+   [yaw.world :as world]
+   [yaw.reaction :as react]
+   [yaw.render :as render]))
+
+(def +myctrl+ (world/start-universe!))
+
+;;; =====================
+;;; The state part
+;;; =====================
+
+(react/register-state  ::cube-state nil)
+
+(def init-cube-state
```

Comment:
Maybe this should be just before the `::initialize` event handler, or even put directly ...
(alternatively, this could be passed as an argument to `register-state`